src/azure-cli/azure/cli/command_modules/acs/_validators.py

```diff
@@ -73,6 +75,22 @@ def validate_create_parameters(namespace):
         raise CLIError('--dns-prefix has no value')
 
 
+def validate_ip_ranges(namespace):
+    if not namespace.api_server_authorized_ip_ranges:
+        return
+
+    allow_all_traffic = "0.0.0.0/32"
+    for ip in namespace.api_server_authorized_ip_ranges.split(','):
+        if ip == allow_all_traffic:
```

Comment:
if the user inputs "0.0.0.0/32" then this should be the only IP address, otherwise we should fail. Can we add a check that the len of result of split(',') should be only 1 or else error. We should not continue on L85.

Reply:
Got it - didn't realize that but it makes sense. Added a check / error message